src/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.md

```diff
@@ -16,7 +16,7 @@ Frontend developers can use this guide to create custom themes to tailor the Mag
 
 You can apply these levels of customization to your site, where the levels require different development skills:
 
-*  You can make relatively simple changes to your site using cascading style sheets (CSS) to change colors and the look-and-feel of various interface components, replacing images, and other relatively cosmetic changes. For more advanced styling, familiarity with LESS (Leaner Style Sheets) and XML (Extensible Markup Language) is helpful.
+*  You can make relatively simple changes to your site using Cascading Style Sheets (CSS) to change colors, and the look-and-feel of various interface components, replacing images, and other relatively cosmetic changes. For more advanced styling, familiarity with LESS (Leaner Style Sheets) and XML (Extensible Markup Language) is helpful.
```

Comment:
In English we don't put a comma before "and" unless the preceding part of the sentence is a list.
```suggestion
*  You can make relatively simple changes to your site using Cascading Style Sheets (CSS) to change colors and the look-and-feel of various interface components, replacing images, and other relatively cosmetic changes. For more advanced styling, familiarity with LESS (Leaner Style Sheets) and XML (Extensible Markup Language) is helpful.
```

Reply:
Done!